tests/end2end/playwright/requests-metadata.spec.js

```diff
@@ -1,13 +1,18 @@
 // @ts-check
 import { test, expect } from '@playwright/test';
 
+/**
+ * Playwright Page
+ * @typedef {import('@playwright/test').APIResponse} APIResponse
+ */
+
 const url = 'index.php/view/app/metadata';
 
 /**
  * Check for a JSON response about the metadata
- * @param {Response} response The response object
+ * @param {APIResponse} response The response object
  *
- * @return {JSON} The JSON response
+ * @return {Promise<any>} The JSON response
```

Comment:
```suggestion
 * @return {Promise<JSON>} The JSON response
```
?

Reply:
`JSON.parse` returns `any` and not `JSON`. It could be an object, an array, a numer or a string.